pyproject.toml

```diff
@@ -78,11 +79,21 @@ build-backend = "poetry.core.masonry.api"
 remove-all-unused-imports = true
 ignore-init-module-imports = true
 ignore-pass-after-docstring = true
+in-place = true
+recursive = true
 
 [tool.isort]
-py_version = 38
-profile = "wemake"
+py_version = 39
+profile = "black"
 line_length = 92
+known_first_party = ["qat"]
+
+[tool.black]
+line-length = 92
+target-version = ["py38", "py39", "py310"]
```

Comment:
super minor: I believe black can infer which python version from project settings, if not it does it per file basis